mime_test.go

```diff
@@ -0,0 +1,74 @@
+package mime2extension
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtToMime(t *testing.T) {
+	var err error
+	var mime string
+
+	err, mime = Lookup(".mp4")
+	makeAssertion(t, err, nil, mime, "video/mp4")
+
+	err, mime = Lookup("/path/to/file.txt")
+	makeAssertion(t, err, nil, mime, "text/plain")
+
+	err, mime = Lookup("file.txt")
+	makeAssertion(t, err, nil, mime, "text/plain")
+
+	err, mime = Lookup(".TXT")
+	makeAssertion(t, err, nil, mime, "text/plain")
+
+	err, mime = Lookup("htm")
+	makeAssertion(t, err, nil, mime, "text/html")
+
+	err, mime = Lookup("foo")
+	makeAssertion(t, err, errors.New("Not found"), mime, "")
+
+	fmt.Println(Extensions("video/mp4"))  // => [mp4 mp4v mpg4]
```

Comment:
were these meant to be assertions?

Reply:
oops, must've left these in after i finished converting them to unit tests